styles.css

```diff
@@ -0,0 +1,421 @@
+:root {
+  --gray: rgba(3, 3, 3, 0.4);
+  --black: rgb(0, 0, 0);
+  --white: rgb(255, 255, 255);
+  --red: #f72210;
+  --roboto: 'Roboto Condensed', sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  width: 100%;
+  background-color: #f4f5f6;
+  font-family: var(--roboto);
+}
+
+.nav-2 img {
+  width: 18px;
+  height: 18px;
+}
+
+.nav-2 input {
+  border: none;
+  width: 60px;
+  outline: none;
+}
+
+.nav-2 {
+  width: 94%;
+}
+
+.h3 {
+  line-height: 24px;
+}
+
+.nav-2 li {
+  padding-left: 10px;
+  padding-right: 6px;
+  width: fit-content;
+}
+
+.nav-3 li {
+  padding-left: 20px;
+}
+
+.font-small {
+  font-size: 16px;
+}
+
+.relative {
+  position: relative;
+}
+
+.bold-700 {
+  font-weight: 700;
+}
+
+.bold-100 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 100;
+}
+
+.bold-400 {
+  font-weight: 400;
+}
+
+.red-bg {
+  background-color: var(--red);
+}
+
+.text-red {
+  color: var(--red);
+}
+
+.font-sm {
+  font-size: 11px;
+}
+
+.gray {
+  color: var(--gray);
+}
+
+.light-gray-bg {
+  background-color: #f4f5f6;
+}
+
+.black {
+  color: var(--black);
+}
+
+.white {
+  color: white;
+}
+
+.subscribe-btn {
+  background-color: #ffe400;
+  border-radius: 24px;
+  font-size: 13px;
+  line-height: 14px;
+  padding: 4px 7px;
+  max-width: 150px;
+}
+
+.weather-img img {
+  height: 20px;
+  width: 20px;
+}
+
+img[alt="user"] {
+  width: 2rem;
+  height: 2rem;
+}
+
+img[alt="menu"] {
+  height: 18px;
+}
+
+img[alt="logo"] {
+  width: 132px;
+  height: 20px;
+}
+
+.small-title {
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.news-title {
+  font-family: var(--roboto);
+  font-size: 22px;
+  line-height: 24px;
+  font-weight: 700;
+}
+
+.news-title a {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.h3 a {
+  color: var(--black);
+  text-decoration: none;
+}
```

Comment:
Group selectors with the same style definition.

Reply:
Implemented this